survey/post-event-survey.yml

```diff
@@ -0,0 +1,128 @@
+    in the future and, more generally, it will hold us accountable! We want to make
+    Maptime Sacramento the best it can be.
+  email_blocked_senders: []
+  email_responses: 1
+  enable_qa: false
+  enable_signatures: false
+  hide_responder_names: false
+  iframe_skip_preview: false
+  payment_amount_cents: 
+  require_payment: false
+  show_advanced_ratings: false
+  show_askers: true
+  show_respondents: true
+  statuses_visible_to_responders: false
+  summary: ''
+response_fields:
+- label: What do you want to learn?
+  field_options:
+    options:
+    - label: Cartography
+      checked: false
+    - label: Programming basics
+      checked: false
+    - label: Web basics (HTML, CSS, Javascript)
+      checked: false
+    - label: Web map APIs (Leaflet, OpenLayers, etc.)
+      checked: false
+    - label: Git and Github basics
+      checked: false
+    - label: Geospatial concepts
+      checked: false
+    - label: Data management and analysis
+      checked: false
+    include_other_option: true
+  required: true
+  blind: false
+  admin_only: false
+  type: ResponseFieldCheckboxes
+- label: What did we do well today?
+  field_options: {}
+  required: true
+  blind: false
+  admin_only: false
+  type: ResponseFieldParagraph
+- label: What could we have done better?
+  field_options: {}
+  required: true
+  blind: false
+  admin_only: false
+  type: ResponseFieldParagraph
+- label: What would you like to see more of?
+  field_options: {}
+  required: true
+  blind: false
+  admin_only: false
+  type: ResponseFieldParagraph
+- label: How was the meeting space?
+  field_options:
+    options:
+    - label: Great
+      checked: false
+    - label: Adequate
+      checked: false
+    - label: Poor
+      checked: false
+    include_other_option: true
+  required: true
+  blind: false
+  admin_only: false
+  type: ResponseFieldRadio
+- label: How should we communicate with you?
+  field_options:
+    options:
+    - label: Meetup
+      checked: false
+    - label: Twitter
+      checked: false
+    - label: Mailing list
+      checked: false
+    - label: Blog
+      checked: false
+    include_other_option: true
+  required: true
+  blind: false
+  admin_only: false
+  type: ResponseFieldCheckboxes
+- label: Is there something you would like to teach at a future Maptime?
+  field_options: {}
+  required: false
+  blind: false
+  admin_only: false
+  type: ResponseFieldParagraph
+- label: Your name
+  field_options:
+    description: Only if you want us to contact you about teaching.
+  required: false
+  blind: false
+  admin_only: false
+  type: ResponseFieldText
+- label: Your email address
+  field_options:
+    description: Only if you want us to contact you about teaching.
+  required: false
+  blind: false
+  admin_only: false
+  type: ResponseFieldEmail
+rating_fields: []
+labels:
+- name: Missing information
+  color: 3600ff
+- name: Questionable
+  color: '336699'
+statuses:
+- name: Reviewed by co-organizers
```

Comment:
Wouldn't it be an issue if this is the only/default status? e.g. how could an application be "reviewed by co-organizers" immediately after it was received?